.goreleaser.yml

```diff
@@ -7,12 +7,14 @@ builds:
       - darwin
       - linux
     goarch:
-      # sops is only available for amd64
       - amd64
+      - arm64
 
 archives:
   - id: zips
     name_template: "{{ .ProjectName }}_{{ .Os }}"
+  - id: newzips
+    name_template: "{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}"
```

Comment:
I still get a failed build:

```
goreleaser --snapshot --skip-publish --rm-dist


   ⨯ release failed after 3.22s error=archive named dist/sopstool_darwin.tar.gz already exists. Check your archive name template
```

Reply:
Note: this happens in travis... for some reason the goreleaser command is in `after_success` which does not affect the build

https://app.travis-ci.com/github/Ibotta/sopstool/builds/248908298#L402

Reply:
I got this locally too, even after removing the new array entry.  I'm not sure what's causing it if I remove my changes, however the fixes you made have resolved it!

Reply:
I think the only thing is this _does_ break the old convention. We might want to major bump and put a note in the README

or, in the 'deploy' shell script, we should add those. but they'd still be missing from the release artifacts in github.  In short, goreleaser doesn't support this edge case where we have a specific and non-specific build case. Unless we had two goreleaser configs.

Reply:
Added a note to the README about the 1.0.0 release breaking the convention.